x-pack/plugins/canvas/shareable_runtime/index.ts

```diff
@@ -6,7 +6,7 @@
  */
 
 export * from './api';
-import '@kbn/core/server/core_app/assets/legacy_light_theme.css';
+import '@kbn/core-apps-server-internal/assets/legacy_light_theme.css';
```

Comment:
Updates path to assets

Reply:
mind if we add the `ci:build-canvas-shareable-runtime` label to test this?

Reply:
I don't mind.

Reply:
@jbudz after adding the flag, the ci run fails on "canvas shareable runtime" for "total size" is 192390 over the configured limit of 8200000"

Reply:
LGTM.  The runtime isn't built in PR's anymore by default because there it adds 3-4 minutes to build times and doesn't have any tests.  The webpack config doesn't change too often so we were okay with the tradeoff.

Reply:
Limit looks like it's something for ops to triage, that's about expected.